.github/workflows/e2e_api_tests.yml

```
@@ -176,47 +173,149 @@ jobs:
+                  timeout_minutes: 5
+                  max_attempts: 2
+                  retry_on: error
+                  command: |
+                      cd tests/pw
+                      npm run start:env
+
+            # Get Playwright version
+            - name: Get installed Playwright version
+              id: playwright-version
+              working-directory: tests/pw
+              run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:126: Double quote to prevent globbing and word splitting

(shellcheck)